src/app/components/Navbar.js

```diff
@@ -0,0 +1,15 @@
+"use client"; 
+import { useState } from "react";
+
+export default function Navbar({ onToggleSidebar }) {
+  return (
+    <nav className="navbar">
+      
+        <div className="brand"><a href="/"><i className="fa fa-droplet"></i> Fluid Tracker</a></div>
+      
+      <button className="burger" onClick={onToggleSidebar}>
+        &#9776;
```

Comment:
```suggestion
        &#9776; {/* burger menu icon */}
```
Probably worth adding a comment explaining what this is ^^'